editor/test/integration/build_test.clj

```diff
@@ -129,7 +129,7 @@
                 :test-fn (fn [pb targets]
                            (is (= {:color [1.0 1.0 1.0 1.0],
                                    :line-break false,
-                                   :scale [1.0 1.0 1.0 0.0],
+                                   :scale [0.0 0.0 0.0 0.0], ; Default from legacy field added by editor.protobuf/field-desc-default. Not in actual Label$LabelDesc.
```

Comment:
`editor.protobuf/field-desc-default` is the function responsible for emitting fake default values for message-type fields in maps returned by `protobuf/read-text`. As stated above, I think we want to get rid of this behavior in the future.